.eslintrc

```diff
@@ -45,6 +45,21 @@
 			{
 				"required": "id"
 			}
-		]
+		],
+		"inclusive-language/use-inclusive-words": [
+			"warn",
+			{
+				"allowedTerms": [ "mastercard" ]
+			}
+		],
+		// this section disables rules causing massive violations in https://github.com/Automattic/woocommerce-payments/pull/1029
```

Comment:
Most of the rules that fail are related to JSDocs. We can either disable them (as you can see, our `.eslintrc` already diverges quite a bit from the default set of rules), or fix them on a new PR. I don't have a strong preference about it.

The other ones are related to React Hooks, and they aren't many. We should fix those.

Reply:
Ok, the react hooks and some unrecognized types (due to missing imports) are topics where I need support.

Let's do the following: I'll go ahead and fix as many JS docs today as I can and we make a pairing session to address the rest. Will this work for you?